test/Atc.Network.Test/Internet/IPPortScanTests.cs

```diff
@@ -0,0 +1,82 @@
+namespace Atc.Network.Test.Internet;
+
+[SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "OK.")]
+[Trait(Traits.Category, Traits.Categories.Integration)]
+[Trait(Traits.Category, Traits.Categories.SkipWhenLiveUnitTesting)]
+public class IPPortScanTests
+{
+    [Fact]
+    public async Task CanConnectWithTcp()
```

Comment:
TestMethod name indicates that it should be able to connect, but the assertions are False in all of the tests - a bit confusing if you ask me.

Same applies for all unittests in the class.